### PR TITLE
feat: new blog post 🔥

### PR DESCRIPTION
**Pull request by promise**

### CODE REVIEW

File: content/blog/2023-07-19-updates-this-summer.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+<!-- truncate -->
+
+## The new `/scores fetch` command
+
+This feature has been requested [since January 2020](https://github.com/countr/countr/issues/16), and we're excited to finally make it available. With the /scores fetch command, you can easily count up all the scores in your counting channel. This feature is perfect for servers that would like to move over to Countr and are not able to export scores from their previous system.
+
+If you want to try it out then go ahead! It responds with a scores file that you can import through `/data import scores`, it will not modify the database directly, so you can safely try it out without worrying about overwriting your scores.
+
+## The new `nodelete` module
```
Comment (wrac4242) [inline content/blog/2023-07-19-updates-this-summer.md:19]: Move this below new reaction action? as it references it before used

File: content/blog/2023-07-19-updates-this-summer.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+This module has been requested [since October 2021](https://github.com/countr/countr/issues/138)! With the "nodelete" module, failed counts will no longer be deleted from the counting channel. This means you can have a normal conversation in your counting channel, if you want that. Combined with the [new react action](#the-new-reaction-action), you can also cross out failed counts.
+
+## The new reaction action
+
+This action has been *highly* requested multiple times, and it's finally here. With the new reaction action, you can react to a count with a Discord emoji or a custom server emoji. For example, you can react with a cross to cross out a failed count, or react with a checkmark to successful counts. Another idea is to add a "nice!" reaction to counts ending in 69.
+
+You can easily stack these as well, so if you want to spell out "nice" with reactions, you can do that!
```
Comment (wrac4242) [inline content/blog/2023-07-19-updates-this-summer.md:27]: combine instead of stack?

**Review by wrac4242 — COMMENTED** [review]
Two small things, one word choice, one on ordering

File: content/blog/2023-07-19-updates-this-summer.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+We're only halfway into the summer, but there have already been a lot of changes to Countr! We're excited to share these changes with you and hope that they enhance your counting experience.
+
+<!-- truncate -->
+
+## The new `/scores fetch` command
+
+This feature has been requested [since January 2020](https://github.com/countr/countr/issues/16), and we're excited to finally make it available. With the /scores fetch command, you can easily count up all the scores in your counting channel. This feature is perfect for servers that would like to move over to Countr and are not able to export scores from their previous system.
+
+If you want to try it out then go ahead! It responds with a scores file that you can import through `/data import scores`, it will not modify the database directly, so you can safely try it out without worrying about overwriting your scores.
```
Comment (djohts) [inline content/blog/2023-07-19-updates-this-summer.md:17]: maybe "overwriting your *current* scores" or something

**Review by djohts — CHANGES_REQUESTED** [review]

File: content/blog/2023-07-19-updates-this-summer.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+# New updates this summer!
+
+We're only halfway into the summer, but there have already been a lot of changes to Countr! We're excited to share these changes with you and hope that they enhance your counting experience.
+
+<!-- truncate -->
+
+## The new `/scores fetch` command
+
+This feature has been requested [since January 2020](https://github.com/countr/countr/issues/16), and we're excited to finally make it available. With the /scores fetch command, you can easily count up all the scores in your counting channel. This feature is perfect for servers that would like to move over to Countr and are not able to export scores from their previous system.
```
Comment (Ta1al) [inline content/blog/2023-07-19-updates-this-summer.md:15]: For consistency, `/scores fetch` should be wrapped in backticks

File: content/blog/2023-07-19-updates-this-summer.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+If you want to try it out then go ahead! It responds with a scores file that you can import through `/data import scores`, it will not modify the database directly, so you can safely try it out without worrying about overwriting your scores.
+
+## The new reaction action
+
+This action has been *highly* requested multiple times, and it's finally here. With the new reaction action, you can react to a count with a Discord emoji or a custom server emoji. For example, you can react with a cross to cross out a failed count, or react with a checkmark to successful counts. Another idea is to add a "nice!" reaction to counts ending in 69.
+
+You can easily combine these as well, so if you want to spell out "nice" with reactions, you can do that!
```
Comment (Ta1al) [inline content/blog/2023-07-19-updates-this-summer.md:23]: Sounds better if it was written as:
> You can easily combine reactions as well,...

but maybe that's just me.

File: content/blog/2023-07-19-updates-this-summer.md (hunk 3 above)
Comment (Ta1al) [inline content/blog/2023-07-19-updates-this-summer.md:17]: This line sounds a bit confusing. Is it supposed to mean that `/scores fetch` will not modify the database, but `/data import scores` will?

**Review by Ta1al — CHANGES_REQUESTED** [review]

**Review by Aditya20-ai — APPROVED** [review]
LGTM

**Review by djohts — APPROVED** [review]
lgtm

**Review by Ta1al — APPROVED** [review]
Nice work!

**Review by lacuna-exe — APPROVED** [review]

File: content/blog/2023-07-19-updates-this-summer.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+
+## The new reaction action
+
+This action has been *highly* requested multiple times, and it's finally here. With the new reaction action, you can react to a count with a Discord emoji or a custom server emoji. For example, you can react with a cross to cross out a failed count, or react with a checkmark to successful counts. Another idea is to add a "nice!" reaction to counts ending in 69.
+
+You can easily combine reactions as well, so if you want to spell out "nice" with reactions, you can do that!
+
+## The new `nodelete` module
+
+This module has been requested [since October 2021](https://github.com/countr/countr/issues/138)! With the "nodelete" module, failed counts will no longer be deleted from the counting channel. This means you can have a normal conversation in your counting channel, if you want that. Combined with the [new react action](#the-new-reaction-action), you can also cross out failed counts.
```
Comment (wrac4242) [inline content/blog/2023-07-19-updates-this-summer.md:27]: maybe remove ", if you want that" and replace with something like "with this module enabled"?

Comment (wrac4242) [inline content/blog/2023-07-19-updates-this-summer.md:27]: "you can also cross out failed counts", bit unclear, maybe change to "you can also react with a cross for failed counts" or something equivalent

File: content/blog/2023-07-19-updates-this-summer.md [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+
+## The new `/scores fetch` command
+
+This feature has been requested [since January 2020](https://github.com/countr/countr/issues/16), and we're excited to finally make it available. With the `/scores fetch` command, you can easily count up all the scores in your counting channel. This feature is perfect for servers that would like to move over to Countr and are not able to export scores from their previous system.
+
+If you want to try it out then go ahead! It responds with a file that you can import later through `/data import scores`. The fetch command will not modify the database directly, so you can safely try it out without worrying about overwriting your current scores.
+
+## The new reaction action
+
+This action has been *highly* requested multiple times, and it's finally here. With the new reaction action, you can react to a count with a Discord emoji or a custom server emoji. For example, you can react with a cross to cross out a failed count, or react with a checkmark to successful counts. Another idea is to add a "nice!" reaction to counts ending in 69.
```
Comment (wrac4242) [inline content/blog/2023-07-19-updates-this-summer.md:21]: specify for the "another idea" that it needs to be a server emoji?

**Review by wrac4242 — COMMENTED** [review]
some slight changing for wording

File: content/blog/2023-07-19-updates-this-summer.md (hunk 6 above)
Comment (Ta1al) [reply content/blog/2023-07-19-updates-this-summer.md:27]: imo `" if you want that"` is better

**Pull request merged**